oslo_vmware/rw_handles.py

```diff
@@ -706,3 +707,50 @@ def close(self):
 
     def __str__(self):
         return "Image read handle"
+
+
+class UrlPullHandle(object):
+    """Base class for a URL from where VMware can download VMDKs
+
+    This kind of handles are being used in conjunction with
+    HttpNfcLeasePullFromUrls_Task, converting an existing lease
+    to "Pull" mode.
+    """
+
+    def url(self):
+        """Returns the URL to the VMDK."""
```

Comment:
I think, this should raise a `NotImplementedError`, as it never returns a valid url.